src/services/impl/ThreadPool.hh

```diff
@@ -11,7 +11,18 @@ namespace impl {
 class ThreadPool : public IThreadPool
 {
 public:
+    // TBD: m_vQueue changed to adapt std::list in-case it threw an exception as it has handlers for it (in VC++ at least). 
```

Comment:
Is this a TBD/TODO? It looks like it's already been done. 

It if has already been done, this comment should be moved next to the definition of `m_vQueue`.